config/manager/kustomization.yaml

```diff
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: eu.gcr.io/kyma-project/telemetry-manager
-  newTag: v20230329-e1d53cad
+  newName: k3d-kyma-registry:5000/telemetry-manager
```

Comment:
As you reverted this change in the previous commit, I assume this should be removed.

Reply:
What exactly do you mean?

Reply:
It was reverted in a follow-up commit

Reply:
[This commit](https://github.com/kyma-project/telemetry-manager/pull/118/commits/c809beacd8d52f2c86519a5fa9924b78037a9dd2) reverted these exact changes.

Reply:
Yup, exactly. So what has to be removed?

Reply:
My bad, it's all good for this change, please ignore my comments above.